src/applications/ivc-champva/10-7959a/chapters/healthInsuranceInformation.js

```diff
@@ -0,0 +1,231 @@
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  phoneUI,
+  phoneSchema,
+  textUI,
+  textSchema,
+  radioUI,
+  radioSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { arrayBuilderPages } from 'platform/forms-system/src/js/patterns/array-builder';
+import { nameWording } from '../../shared/utilities';
+
+const radioLabels = {
+  group: 'Employer sponsored insurance (group)',
+  nonGroup: 'Individual or private insurance (non-group)',
+  medicare: 'Medicare Part A or B (hospital and medical insurance)',
+  other: 'Insurance type not listed',
+};
+
+const hintText =
+  'This includes any coverage through a family member, health insurance from an employer, and supplemental or secondary insurance.';
+
+const yesNoContent = {
+  title:
+    'Does the beneficiary have any other health insurance coverage that isn’t through CHAMPVA?',
+  labels: {
+    Y: 'Yes',
+    N: 'No',
+  },
+  hint: hintText,
+  labelHeaderLevel: '5',
+};
+
+/** @type {ArrayBuilderOptions} */
+const options = {
+  arrayPath: 'policies',
+  nounSingular: 'policy',
+  nounPlural: 'policies',
+  required: true,
+  isItemIncomplete: item => !item?.name && !item?.type, // include all required fields here
+  maxItems: 2,
+  text: {
+    getItemName: item => item.name,
+    cardDescription: item =>
+      item?.type === 'other'
+        ? `${item?.otherType}`
+        : `${radioLabels[(item?.type)]}`,
+    summaryTitle: 'Beneficiary’s health insurance review',
+  },
+};
+
+export const insuranceStatusSchema = {
+  uiSchema: {
+    ...titleUI(({ formData }) => {
+      return `${nameWording(
+        formData,
+        true,
+        true,
+        true,
+      )} health insurance status`;
+    }),
+    hasOhi: {
+      ...yesNoUI({
+        updateUiSchema: formData => {
+          return {
+            'ui:title': `${
+              formData?.certifierRole === 'applicant' ? 'Do' : 'Does'
+            } ${nameWording(
+              formData,
+              false,
+              false,
+              true,
+            )} have health insurance coverage that isn’t through CHAMPVA?`,
+            'ui:options': {
+              hint: hintText,
+            },
+          };
+        },
+      }),
+    },
+  },
+  schema: {
+    type: 'object',
+    required: ['hasOhi'],
+    properties: {
+      titleSchema,
+      hasOhi: yesNoSchema,
+    },
+  },
+};
+
+const policyPage = {
+  uiSchema: {
+    ...arrayBuilderItemFirstPageTitleUI({
+      title: 'Policy information',
+      nounSingular: options.nounSingular,
+    }),
+    name: textUI('Name of insurance provider'),
+    policyNum: textUI('Policy number'),
+    providerPhone: phoneUI('Insurance provider phone number'),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      name: textSchema,
+      policyNum: textSchema,
+      providerPhone: phoneSchema,
+    },
+    required: ['name'],
+  },
+};
+
+const insuranceProviderPage = {
```

Comment:
Second page of list loop - radio buttons with a hidden "other" text field that shows up if the user clicks the "other" option.